src/com/facebook/buck/android/AndroidBinaryGraphEnhancer.java

```diff
@@ -597,7 +600,8 @@ PreDexMerge createPreDexMergeRule(
         allPreDexDeps,
         dexForUberRDotJava,
         dxExecutorService,
-        xzCompressionLevel);
+        xzCompressionLevel,
+        dxConfig.getDxMaxHeapSize());
```

Comment:
Nope.  I just had a few speedbumps on the import.  No other changes required IMO.